### ISSUE TITLE
Add Argument to aws_ssm_association

### ISSUE THREAD

**Issue by jonjohnston** [terraform-providers/terraform-provider-aws#1364]
Hi there,

I have tested and created an association successfully. 
```
resource "aws_ssm_association" "Testing" {
	name = "AWS-ConfigureCloudWatch"
	parameters {
		status = "Enabled"
		properties =  "${data.template_file.testingfile.rendered}"
	}
	targets {
		key = "tag:Name"
		values = ["Test-Association"]
	}
}
```

However, awscli provides 
`aws ssm create-association --name Testing --schedule-expression ` A cron expression when the association will be applied to the target(s).

This does not exist for terraform according to the documentation https://www.terraform.io/docs/providers/aws/r/ssm_association.html. Please add argument schedule_expression

Would look like this:
```
resource "aws_ssm_association" "Testing" {
	name = "AWS-ConfigureCloudWatch"
        schedule_expression = "cron(0 16 ? * TUE *)"
	parameters {
		status = "Enabled"
		properties =  "${data.template_file.testingfile.rendered}"
	}
	targets {
		key = "tag:Name"
		values = ["Test-Association"]
	}
}
```

**Comment by stack72** [comment]
Hi @jonjohnston 

This has been added in #1253

Paul

//cc @radeksimko

**Comment by tombuildsstuff** [comment]
Closing as this has been fixed via #1253
